docs/syntax_and_semantics/literals/array.md

```diff
@@ -80,3 +80,32 @@ The type arguments can be explicitly specified as part of the type name:
 ```crystal
 Set(Int32){1, 2, 3}
 ```
+
+## Splat Expansion
+
+The splat operator can be used inside array and array-like literals to insert multiple values at once.
+
+```crystal
+[1, *coll, 2, 3]
```

Comment:
What does "coll" stand for? :o
Somehow the fact that I can't guess it makes me more confused than if it was a random combination of letters